src/pages/office/cat-cpc/office.cat-cpc.page.html

```diff
@@ -69,7 +82,7 @@ <h4>Test summary</h4>
             </candidate-description>
 
             <identification
-              *ngIf="!(isDelegated && isFail())"
+              *ngIf="!(isDelegated && isPass())"
```

Comment:
These `*ngIf`s are getting pretty complex

Reply:
for idenification, isnt it as simple as if not a delegated test then display? im only looking at the UX not the story detail tho.

Reply:
nice spot andy, addressed, not sure how i could amend the others to be more concise, open to suggestions